packages/utils/package.json

```diff
@@ -32,7 +32,9 @@
     }
   },
   "devDependencies": {
-    "@noble/hashes": "^1.5.0"
+    "@noble/hashes": "^1.5.0",
+    "bignumber.js": "^9.1.2",
+    "date-fns": "2.30.0"
```

Comment:
these should be under `dependencies`. `@noble/hashes` gets bundled so we can keep it here